waiter/src/waiter/core.clj

```diff
@@ -921,8 +924,14 @@
    :token->token-metadata (pc/fnk [[:curator kv-store]]
                             (fn token->token-metadata [token]
                               (sd/token->token-metadata kv-store token :error-on-missing false)))
-   :validate-service-description-fn (pc/fnk [[:state service-description-builder]]
+   :validate-service-description-fn (pc/fnk [[:state authenticator service-description-builder]]
                                       (fn validate-service-description [service-description]
+                                        (let [authentication (or (get service-description "authentication") "standard")
+                                              authentication-providers (into #{"disabled" "standard"} (auth/get-authentication-providers authenticator))]
```

Comment:
If `auth/get-authentication-providers` returns standard, then we do not need to add it in explicitly here.

Reply:
"disabled" and "standard" seem fine here

Reply:
Please refer to comments about `standard` in `composite.clj`